perllib/FixMyStreet/App/Controller/Report.pm

```diff
@@ -183,9 +183,9 @@ sub load_problem_or_display_error : Private {
 sub load_updates : Private {
     my ( $self, $c ) = @_;
 
-    my $updates = $c->model('DB::Comment')->search(
-        { problem_id => $c->stash->{problem}->id, state => 'confirmed' },
-        { order_by => [ 'confirmed', 'id' ] }
+    my $updates = $c->cobrand->updates->search(
```

Comment:
I guess it probably makes sense to make a similar change to Contact.pm (moderation of another cobrand's update) and Photo.pm (showing photo of another cobrand's update if requested)? (And perhaps Questionnaire.pm for parity with here, though don't think that would have any affect anywhere.)